src/util/http/websocket/QueryHub.cpp

```diff
@@ -34,8 +34,8 @@ inline auto makeDeleteFromDistributors =
 net::awaitable<std::shared_ptr<QueryToSocketDistributor>>
 QueryHub::createOrAcquireDistributorInternalUnsafe(QueryId queryId,
                                                    bool isSender) {
-  if (socketDistributors_->contains(queryId)) {
-    auto& reference = socketDistributors_->at(queryId);
+  if (socketDistributors_.get().contains(queryId)) {
```

Comment:
bind the reference `distributors_.get()` to a variable, either locally in this function, or even in class scope, to get rid of the many repeated `get()` calls.
`if (distributors.contains(queryId))`.